papierowyRPG_API/Models/Character.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace papierowyRPG_API.Models
+{
+    public class Character
+    {
+        public int Id { get; set; }
```

Comment:
Let's keep it consistent across all models
```suggestion
        public int ID { get; set; }
```